### PR TITLE
Add content directive for upstream

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: templates/nginx.conf.j2 [hunk 1]
```diff
@@ -115,7 +115,11 @@ http {
 
     {% for upstream in nginx_upstreams | default([]) %}
     upstream {{ upstream.name }} {
+        {% if upstream.path is defined %}
         server {{ upstream.path }};
+        {% else %}
+		{{ upstream.content }}
+		{% endif %}
```
Comment (SlothOfAnarchy) [inline templates/nginx.conf.j2]: Indentation

File: README.md [hunk 2]
```diff
@@ -64,26 +64,31 @@ upstream, maps, and global vars are to be defined by using either
 
 ### Upstream Vars
 `nginx_upstreams` is a list of dicts containing the following two entries.
-| Name   |      Required      | Default | Description                       |
-|:-------|:------------------:|:--------|:----------------------------------|
-| `name` | :heavy_check_mark: |         | Upstream name used in domain_vars |
-| `path` | :heavy_check_mark: |         | url or socket to php listener     |
+
+| Name      | Required           | Default | Description                                                                                                             |
```
Comment (SlothOfAnarchy) [inline README.md:68]: The default column is not really needed anywhere. You could merge Required and Default

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Scriptkiddi) [reply README.md:68]: Yes i could or stick to what this role already has

File: templates/nginx.conf.j2 (hunk 1 above)
Comment (Scriptkiddi) [reply templates/nginx.conf.j2]: fixed

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy please rereview

File: README.md (hunk 2 above)
Comment (SlothOfAnarchy) [reply README.md:68]: Of course, I've just seen you change it on existing roles and create PRs to reflect on better practices before and though you might want to do this here, too.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**